docker-compose.yml

```diff
@@ -1,10 +1,12 @@
-version: "3.8"
+version: '3.8'
 
 services:
   businext_bot:
     image: businext:latest
     build: .
     restart: unless-stopped
+    env_file:
+      - .env
```

Comment:
Might be good to start proactively documenting env variables users need to configure instead of a large scale README change later on down the line

Reply:
Ye, we should document things incrementally as we work on more features. A rough draft README for now so we can remember the things we did more easily.